app/lib/authentication.py

```diff
@@ -4,20 +4,14 @@
 from flask import current_app, abort, request
 
 
-def requires_authentication(view):
-    @wraps(view)
-    def wrapped_view(*args, **kwargs):
-        if current_app.config['AUTH_REQUIRED']:
-            incoming_token = get_token_from_headers(request.headers)
-
-            if not incoming_token:
-                abort(401)
-            if not token_is_valid(incoming_token):
-                abort(403)
-
-        return view(*args, **kwargs)
-
-    return wrapped_view
+def requires_authentication():
```

Comment:
:+1:  Yes I forgot to add this in my branch until it was very nearly finished.  Definitely makes sense to require auth for everything, certainly for the time being.